src/zc/buildout/tests.py

```diff
@@ -3168,7 +3168,8 @@ def test_wheel_to_egg(self):
         [egg_name] = [n for n in os.listdir(self.sample_eggs)
                   if n.startswith('demo-0.3-')]
         path = os.path.join(self.sample_eggs, egg_name)
-        os.rename(path, os.path.join(self.sample_eggs, 'demo-0.3.whl'))
+        os.rename(path, os.path.join(self.sample_eggs,
+                                     'demo-0.3-py2.py3-none-any.whl'))
 
```

Comment:
That's completely wrong! You can't create a wheel by just renaming an egg file! And even if the wheel is valid, you can't install it by just uncompressing it.

Reply:
You're right.

The original os.rename() that I replaced was probably only used to test if some code path was being executed.

And that unpack method that I'm calling: that needs to be another one.